src/components/atoms/Input/Input.tsx

```diff
@@ -209,14 +212,14 @@ export const Input: React.FC<InputProps> = (props) => {
             {...notInputProps}
             id={label ? generateRandomInputId : undefined}
           />
-          {label && (
-            <fieldset>
+          <fieldset>
```

Comment:
this code makes that if label is falsy then you render empty `<fieldset>`, this is intended ?